master/contrib/github_buildbot.py

```diff
@@ -104,10 +105,10 @@ def render_POST(self, request):
             request.setResponseCode(OK)
             return json.dumps({"result": "pong"})
 
-        # Reject non-push, non-ping events
-        if event_type != "push":
+        # Reject events not involving code changes
+        if event_type not in ("push", "pull_request"):
```

Comment:
an enum for even_type may be welcome
https://docs.python.org/3/library/enum.html